src/xr/constants.js

```diff
@@ -149,3 +149,35 @@ export const XRTRACKABLE_PLANE = 'plane';
  * @description Mesh - indicates that the hit test results will be computed based on the meshes detected by the underlying Augmented Reality system.
  */
 export const XRTRACKABLE_MESH = 'mesh';
+
+/**
+ * @constant
+ * @type string
+ * @name XRDEPTHSENSINGUSAGE_CPU
+ * @description CPU - indicates that depth sensing preferred usage is CPU. This usage path is guaranteed to be supported.
+ */
+export const XRDEPTHSENSINGUSAGE_CPU = 'cpu-optimized';
+
+/**
+ * @constant
+ * @type string
+ * @name XRDEPTHSENSINGUSAGE_GPU
+ * @description GPU - indicates that depth sensing preferred usage is GPU.
+ */
+export const XRDEPTHSENSINGUSAGE_GPU = 'gpu-optimized';
+
+/**
+ * @constant
+ * @type string
+ * @name XRDEPTHSENSINGDATAFORMAT_LUMINANCEALPHA
+ * @description Luminance Alpha - indicates that depth sensing preferred raw data format is Luminance Alpha. This format is guaranteed to be supported.
+ */
+export const XRDEPTHSENSINGDATAFORMAT_LUMINANCEALPHA = 'luminance-alpha';
```

Comment:
Can you just reuse `PIXELFORMAT_L8_A8` and `PIXELFORMAT_R32F` instead of defining new constants? And if not, these constant are super-verbose. 40 characters for this one! `XRDEPTHSENSINGFORMAT_L8A8` and `XRDEPTHSENSINGFORMAT_F32` maybe?

Reply:
The values are provided by WebXR Depth Sensing Specs, so reuse of GL constants won't be possible here.
Renamed them to shorter versions.